server/controllers/userInputController.ts

```diff
@@ -36,6 +40,10 @@ const createUserInputController = (
   ) => {
     try {
       const userId = res.locals.userId;
+      // Check for authorization header. If not provided, throw 401 error
+      if (!req.headers.authorization) {
```

Comment:
The check for auth header in this controller is redundant here as it is already checked in the auth middleware.